courses-app/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,55 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import {LoginComponent} from "./shared/login/login.component";
+import {RegistrationComponent} from "./shared/registration/registration.component";
+import {CoursesComponent} from "./features/courses/courses.component";
+import {CourseComponent} from "./features/course/course.component";
+import {CourseFormComponent} from "./shared/course-form/course-form.component";
+import {AuthorizedGuard} from "./auth/guard/authorized.guard";
+import {AdminGuard} from "./auth/guard/admin.guard";
+import {NotAuthorizedGuard} from "./auth/guard/not-authorized.guard";
+import {authorizedCanActivateGuard} from "./auth/guard/authorizedCanActivate.guard";
```

Comment:
In the task should be only 3 types of guard: Admin, NotAuthorized and Authorized.